docs/getting-started/integrate/node.mdx

```diff
@@ -35,20 +35,19 @@ export const server = setupServer(...handlers)
 
 ## Setup
 
-It's recommended to configure API mocking as a part of your tests teardown, so that your tests don't have to reference any mocking during their runs, focusing on testing what matters.
+It's recommended to configure API mocking as a part of your tests setup, so that your tests don't have to reference any mocking during their runs, focusing on testing what matters.
 
 ### Using Create React App
 
-In Create React App there is a `src/setupTests.js` module that is used as a Jest teardown configuration.
+In Create React App there is a `src/setupTests.js` module that is used as a Jest [setupFilesAfterEnv](https://jestjs.io/docs/en/configuration.html#setupfilesafterenv-array) configuration.
```

Comment:
With the correction this sentence is invalid. Maybe let's consider

```
In Create React App there is a `src/setupTests.js` module that is used in the Jest configuration as the [setupFilesAfterEnv]() value.
```

What do you think?

Reply:
I think you're right. Thank you for the suggestion. Updated.